src/Crawler.php

```diff
@@ -301,9 +338,14 @@ protected function convertBodyToString(StreamInterface $bodyStream, $readMaximum
         return $body;
     }
 
+    protected function createRobotsTxt(UriInterface $uri): RobotsTxt
+    {
+        return RobotsTxt::create($uri->withPath('/robots.txt'));
+    }
+
     /**
      * @param ResponseInterface|null $response
-     * @param CrawlUrl $crawlUrl
+     * @param CrawlUrl               $crawlUrl
```

Comment:
I don't like this indentation, but I guess StyleCI will take care of this.

Reply:
I've disabled it in my auto formatter :)